pages/osx/tart.md

```diff
@@ -0,0 +1,36 @@
+# tart
+
+> Build, run and manage macOS and Linux virtual machines (VMs) on Apple Silicon.
+> More information: <https://github.com/cirruslabs/tart>.
+
+- Pull a remote VM image:
+
+`tart pull {{acme.io/org/name:tag}}`
+
+- Clone a VM from a local or remote image source:
+
+`tart clone {{source-vm}} {{vm-name}}`
+
+- Create a new Mac VM from an ipsw file:
+
+`tart create --from-ipsw={{[latest|path/to/file.ipsw]}} {{vm-name}}`
+
+- Run an existing VM:
+
+`tart run {{vm-name}}`
+
+- Run an existing VM with a mounted directory:
+
+`tart run --dir={{directory}}:{{/path/to/local/directory}} {{vm-name}}`
```

Comment:
```suggestion
- Run an existing VM with a specific mounted directory:

`tart run --dir={{path/to/directory}}:{{/path/to/local_directory}} {{vm-name}}`
```

Mandatory fix: [path format is incorrect](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).